components/homepage/SkillIcon.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC, useState } from 'react';
+import Image from 'next/image';
+import { Tooltip } from '@nextui-org/react';
+
+interface SkillIconProps {
+  src: string;
+  text: string;
+}
+
+const SkillIcon: FC<SkillIconProps> = ({ src, text }) => {
+  const [colourIcon, setColourIcon] = useState<boolean>(false);
+
+  return (
+    <Tooltip content={text} placement='bottom'>
+      <div
+        key={src}
+        className={`cursor-pointer rounded-full ${
+          colourIcon
+            ? 'z-10 bg-white900 shadow-lg dark:bg-black300'
+            : 'bg-white800'
+        }  p-3.5 dark:bg-black300 lg:p-6`}
```

Comment:
can't you just do this with tailwind class utilities instead of introducing react state ?
```suggestion
        className='cursor-pointer rounded-full hover:z-10 hover:bg-white900 hover:shadow-lg dark:hover:bg-black300 bg-white800 p-3.5 dark:bg-black300 lg:p-6'
```

Reply:
That's much simpler! :D thanks